functions.php

```diff
@@ -64,8 +64,8 @@ function betterChildBreadcrumbs() {
 		$childBreadcrumb = $startLink.$pageLink.$endLink.$pageTitle.$closeLink;
 	}
 
-	if ( $parentBreadcrumb != '' ) {echo '<span>'.$parentBreadcrumb.'</span>';}
-	if ( $childBreadcrumb != '' ) {echo '<span>'.$pageTitle.'</span>';}
+	if ( '' !== $parentBreadcrumb ) {echo '<span>'.$parentBreadcrumb.'</span>';}
+	if ( '' !== $childBreadcrumb ) {echo '<span>' . esc_html( $pageTitle ) . '</span>';}
```

Comment:
Throughout this PR, I'm running various title variables through `esc_html()` on the understanding that titles won't themselves hold HTML tags. This seems pretty safe, but I wanted to mention it explicitly.